### DISCUSSION TITLE
Keeping sync with AWS Quickstart

### DISCUSSION THREAD

**Discussion by s0enke** [superwerker/superwerker#180] [General] [answered]
## Situation:

 - The AWS QuickStart has no version information. It's not possible to determine which version has been installed. 
 - The superwerker repo is synced to the AWS QuickStart repo manually from time to time, a manual process with usually cherry picking.

## Solution proposal:

- After each superwerker release, we create a PR named `superwerker-release-x.y.z` at the AWS QuickStart with the generated files from the release process (which already contain the version information). 
- We only sync the `templates` folder from superwerker to the AWS QuickStart (list can be extended later on). 
- We ignore everything else

Out of scope:
- There is no automatism for back merging from the AWS QuickStart, since edits in the AWS QuickStart are rare (e.g. documentation/wording) after the initial release.

TODO:
- [x] Fix environment variable replacement for PR body ( https://github.com/superwerker/releasetests/blob/main/.github/workflows/release.yml ) (we did not know how to do it so we hardcoded the vaues for now)
- [x] ~~ADR if accepted~~
- [x] Create GitHub bot user to create PR in the AWS quickstart repository
- [ ] Put it into production

**Comment by kenhorn** [comment]
Think a clean way of determining the current deployed version is important. I arguably rushed into deployment without understanding the whats-next steps in sufficient detail. Having a path to upgrades / maintain best practice -- or make it clear it could be / is, a one-time operation -- is important for adoption. I'm reticent to use the configuration in production without further understanding of the path forward -- or indeed to customise parts without understanding the impact on ability to take updates.

**Comment by s0enke** [comment, marked as answer]
We think we finished this one and now a PR is created automatically at the AWS Quickstart for each superwerker release, example: https://github.com/aws-quickstart/quickstart-superwerker/pull/28. It also includes the version information now when installing it via AWS QuickStart.
